config/puma.rb

```diff
@@ -1,28 +1,30 @@
+# frozen_string_literal: true
```

Comment:
there are some files we don't want to modify for example `bin/`, `config/`, `schema.rb`. You can specify files to ignore in `.rubocop.yml`. Just put into this file following snippet of code:
```
AllCops:
  DisplayCopNames: true
  TargetRubyVersion: 2.5 # chose proper version
  Exclude:
    - 'bin/*'
    - 'lib/tasks/*'
    - 'script/*'
    - 'vendor/**/*'
    - 'node_modules/**/*'
    - 'db/migrations/*'
    - 'db/schema.rb'
    ```
    
    In this file you can specify your own rules but if you don't know what should be done you can use [github rubocop](https://github.com/github/rubocop-github)